ios/sdk/src/callkit/JMCallKitProxy.swift

```diff
@@ -47,6 +46,16 @@ import Foundation
         }
     }
     
+    /// Enables the proxy in between callkit and the consumers of the SDK
+    /// Default to enabled, set to false when you don't want to use callkit
+    @objc public static var callKitProxyEnabled: Bool = true {
```

Comment:
since this is part of the JMCallKitProxy class, can we just call it `enabled` ? We know exactly what this is about.